streams/shared/src/main/scala/zio/stream/ZStreamOps.scala

```diff
@@ -0,0 +1,13 @@
+package zio.stream
+
+import scala.reflect.ClassTag
+
+trait ZStreamOps {
+  implicit class Syntax[R, E, O](self: ZStream[R, E, O]) {
```

Comment:
You can just make this an implicit class inside the `ZStream` companion object.

Reply:
Ok. Wasn't sure I should touch the companion object. Will do

Reply:
Moved it to the companion object